content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md

```diff
@@ -1,9 +1,203 @@
 ---
 title: "Pipeline Settings"
-keywords: 'KubeSphere, Kubernetes, Docker, Jenkins, pipelines'
+keywords: 'KubeSphere, Kubernetes, Docker, Jenkins, Pipelines'
 description: 'Understand various pipeline properties in a DevOps project.'
 linkTitle: "Pipeline Settings"
 weight: 11280
 ---
 
-TBD.
\ No newline at end of file
+When you create a pipeline, you can customize its configurations through various settings. This document illustrates the settings when creating a pipeline.
```

Comment:
This document illustrates the settings when creating a pipeline.
->
This document illustrates these settings in detail.